Templates (Project)/Hamburger/Views/Shell.xaml

```diff
@@ -52,8 +52,10 @@
             </Controls:HamburgerButtonInfo>
             <!--  settingspage button  -->
             <Controls:HamburgerButtonInfo x:Name="SettingsButton"
-                                          PageParameter="0"
```

Comment:
I don't want to add an unused parameter. I realize that 0 is the default if none is passed, but as part of the template, I want to demonstrate empty parameters and handling them. I hope this makes sense.